### PR TITLE
Create loadtest environment

### PR DESCRIPTION
**Pull request by RMcVelia**
### Context

New loadtest environment for find in bat-prod space
- same infrastructure as prod
- manual deploy via deploy workflow or make (not automated)

### Changes proposed in this pull request

- Add new env to Makefile
- Add to deploy workflow 
- terraform updates
- Add required rails configuration files

Also added outside of this PR
- required gh secrets
- AZ KV secrets
- AZ storage container

### Guidance to review

- [x] confirm KV app/infra secrets are as expected for a loadtest
- [x] confirm infrastructure requirements
- [ ] if dfe-signin required url's will need to be added to dfe-signin
- [ ] deploy-plan loadtest

### Checklist

- [ ] Rebased `main`
- [ ] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

**Review by smneal — APPROVED** [review]
LGTM - pending check of the secrets

**Comment by smneal** [comment]
- Key Vault secrets are a subset of the production secrets.
- Infrastructure sizing has been updated to align with production

**Comment by smneal** [comment]
Discussed with @SamJamCul and @saliceti and updated the relevant secrets in Key Vault.

File: config/settings/loadtest.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+base_url: https://find-loadtest.london.cloudapps.digital
+#assets_url: https://loadtest-assets.london.cloudapps.digital
+apply_base_url: https://apply-loadtest.london.cloudapps.digital
+teacher_training_api:
+  base_url: https://publish-teacher-training-loadtest.london.cloudapps.digital
+valid_referers:
+  - https://find-loadtest.london.cloudapps.digital
+
+# from the previous loadtest setup
+# is this still valid
```
Comment (SamJamCul) [inline config/settings/loadtest.yml:10]: Yeah, seems to be in use, and we don't want to turn it on

**Review by SamJamCul — APPROVED** [review]

**Pull request merged**